lib/ting.rb

```diff
@@ -50,15 +50,30 @@ def camelize(str)
       )
     end
 
+    # The longest syllables are six letters long (chuang, shuang, zhuang).
+    SYLLABLE_REGEXP = /[A-Za-züÜ]{1,6}\d?/
```

Comment:
I changed the upper bound from 5 to 7 characters in #9, but that was based on the assumption that `pretty_tones` would also parse erhua (e.g. zhuangr) at some point. But since `pretty_tones` passes the syllables on to a `Ting.reader` that doesn't understand erhua, I've reduced the limit to 6.